docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -379,6 +379,79 @@ I will also verify that the household
+    * If there is not a relative of the appropriate age available, assign the reference person 
+- Child is non-relative (roommate or other nonrelative) to reference person 
+    * Assign another non-relative of the reference person who is between 20 and 45 years older than the child. If there are multiple, select one of the same race/ethnicity. If there are multiple with the same race/ethnicity, assign at random.  
+    * If there is not a non-relative of the appropriate age available, assign to a non-relative of any age (select at random if multiple) 
+    * If there are not any other non-relatives in the house, make "N/A"
+- Child is the reference person 
+    * Assign a parent, if available 
+    * Otherwise, assign another relative who is between 20 and 45 years older than the child. If there are multiple, select one of the same race/ethnicity. If there are multiple of the same race/ethnicity, assign at random.
+    * If there are no other relatives in the house, make "N/A"
+
+Once the parent/guardian is assigned, if there is a spouse or unmarried partner 
+for that simulant (reference person and spouse/unmarried partner ONLY), then 
+include both as parents/guardians. Otherwise only include the one as a parent/guardian. 
+
+**For a simulant who is below 24 and in GQ at college:**
+
+Simulant will be randomly assigned to a parent/guardian based on the below rules: 
+
+- 78.5% will be assigned to a parent/guardian within their state. The remainder will be assigned out of state source1_  
+- Match to a person 20 to 45 years older than the child 
+- If child is not "Multiracial or Some Other Race", match parent's race. If child is "Multiracial or Some Other Race", then assign to a parent of any race
+- Assign to reference people source2_ 
+    * 23% female reference people without a listed spouse 
+    * 5% male reference people without a listed spouse 
+    * Remainder to people with spouses, include both parents 
+
+
+.. _source1: https://www.statista.com/statistics/236069/share-of-us-students-who-enrolled-in-a-college-in-their-own-state/ 
+
+.. _source2: https://nces.ed.gov/programs/coe/indicator/cce/family-characteristics 
+
+
+**Limitations**
+
+#. The foster care system is complex. We have the foster kid assigned within the house they are currently living. If we model the foster care system in more detail, we might improve this at some point. 
+#. We have "parents" fall between 20-45 older than the child. This is an oversimplification. Some parents (especially men) fall outside of this range. Also some age gaps are more common than others. 
+#. The only people who are seen as "in college" are in GQ in college. Plenty of people attend college from home, but we do not track occupation so are not accounting for this. 
```

Comment:
We do track employer, but I think it might get complicated. For example, how do we separate a U of Washington employee from a U of Washington student? We would probably need some kind of special "student" tag separate from employer. We could consider adding this to the model though if it is important.

Reply:
And, people would work and attend school at the same time. I'd change this sentence to say "education" instead of "occupation" to reflect that we would model it separately.